spec/facility_spec.rb

```diff
@@ -2,25 +2,266 @@
+    @facility_2 = Facility.new({name: 'DMV Northeast Branch', address: '4685 Peoria Street Suite 101 Denver CO 80239', phone: '(720) 865-4600'})
+    @cruz = Vehicle.new({vin: '123456789abcdefgh', year: 2012, make: 'Chevrolet', model: 'Cruz', engine: :ice} )
+    @bolt = Vehicle.new({vin: '987654321abcdefgh', year: 2019, make: 'Chevrolet', model: 'Bolt', engine: :ev} )
+    @camaro = Vehicle.new({vin: '1a2b3c4d5e6f', year: 1969, make: 'Chevrolet', model: 'Camaro', engine: :ice} )
   end
+
   describe '#initialize' do
     it 'can initialize' do
-      expect(@facility).to be_an_instance_of(Facility)
-      expect(@facility.name).to eq('DMV Tremont Branch')
-      expect(@facility.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
-      expect(@facility.phone).to eq('(720) 865-4600')
-      expect(@facility.services).to eq([])
+      expect(@facility_1).to be_an_instance_of(Facility)
+      expect(@facility_1.name).to eq('DMV Tremont Branch')
+      expect(@facility_1.address).to eq('2855 Tremont Place Suite 118 Denver CO 80205')
+      expect(@facility_1.phone).to eq('(720) 865-4600')
+      expect(@facility_1.services).to eq([])
     end
   end
 
   describe '#add service' do
     it 'can add available services' do
-      expect(@facility.services).to eq([])
-      @facility.add_service('New Drivers License')
-      @facility.add_service('Renew Drivers License')
-      @facility.add_service('Vehicle Registration')
-      expect(@facility.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+      expect(@facility_1.services).to eq([])
+      @facility_1.add_service('New Drivers License')
+      @facility_1.add_service('Renew Drivers License')
+      @facility_1.add_service('Vehicle Registration')
+      expect(@facility_1.services).to eq(['New Drivers License', 'Renew Drivers License', 'Vehicle Registration'])
+    end
+  end
+
+describe '#check cruz.registration_date' do
+  it 'checks registration date of cruz' do
+    expect(@cruz.registration_date).to eq(nil)
+  end
+end
+
+  describe '#check facility_1(F1) vehicles' do
+    it 'checks registered_vehicles for facility_1' do
+      expect(@facility_1.registered_vehicles).to eq([])
+    end
+  end
+
+  describe '#check F1 collected fees' do
+    it 'checks collected fees for facility_1' do
+      expect(@facility_1.collected_fees).to eq(0)
+    end
+  end
+
+  describe '#register cruz and check info' do
+    it 'registers cruz and checks info' do
+      expect(@cruz).to be_an_instance_of(Vehicle)
+      @facility_1.add_service('Vehicle Registration')
+      @facility_1.register_vehicle(@cruz)
+      expect(@cruz.registration_date).to eq(Date.today)
+      expect(@cruz.plate_type).to eq(:regular)
+      #require the facility to have the service you're performing
+    end
+  end
+
+  #decided not to make a test for my helper method `set_plate_type` 
+  #because those tests are relevant and included in my other tests
```

Comment:
Fine for this project, though in the future we will ask you to test all helper methods. What you have covering the `set_plate_type` behavior are **integration** tests; testing `set_plate_type` in isolation would be a **unit** test.